src/test/utils/VaultUtils.sol

```diff
@@ -230,6 +230,42 @@ abstract contract BasicVaultTest is UniswapV2Fixture {
         vault.nextEpoch(reserves0, reserves1 * 2);
     }
 
+    function test_dontDepositLessThanMIN_LPt0() public {
+        depositToken0();
+        depositToken1();
+
+        advance();
+
+        assertGt(getToken0Active(), 0);
+        assertGt(getToken1Active(), 0);
+
+        uint256 token0BalanceDay0 = vault.token0BalanceDay0(address(this));
+        vault.withdrawToken0(token0BalanceDay0 - 100);
```

Comment:
can we query `vault.MIN_LP()` ? Or otherwise indicate what this magic number is

Reply:
for the `100` I mean

Reply:
its a little tricky, because we don't know how many tokens we are going to end up with after the end of the epoch
but maybe we can do `vault.MIN_LP()/10` with the assumption that we are not going to see a 10x increase in balance